tools/publish-jar-local.sh

```diff
@@ -0,0 +1,52 @@
+set -euo pipefail
+
+if [[ -f "$HOME/.jabba/jabba.sh" ]]; then
+  echo "Found jabba"
+  source "$HOME/.jabba/jabba.sh"
+  jabba use adopt@1.8.0-292
+fi
+
+JAVA_VERSION=`java -version 2>&1 | awk -F '"' '{print $2}'`
+if [[ $JAVA_VERSION != "1.8.0_292" ]]; then
```

Comment:
I would better check only major and minor, the patch is not necessary. It makes it very restrictive.